picard/ui/metadatabox/__init__.py

```diff
@@ -137,6 +137,13 @@ def get_tag_name(self, index):
 
 class MetadataBox(QtWidgets.QTableWidget):
 
+    MIMETYPE_PICARD_TAGS = "application/vdr.picard"
```

Comment:
The problem I have with copying only to these two types is that `vdr.picard` is by definition only going to be understandable by picard, and there is therefore no structured form of data that can be used for pasting by e.g. a spreadsheet or a database.

I would personally like to see a Copy being sent to the clipboard as mimetypes for JSON, for TSV and for CSV (perhaps with and without headers if there are different standard mimetypes for these), so that receiving applications have a wider choice of structured data to choose from.

Reply:
> The problem I have with copying only to these two types is that `vdr.picard` is by definition only going to be understandable by picard, and there is therefore no structured form of data that can be used for pasting by e.g. a spreadsheet or a database.

Can you give exact use cases, application names and mime-types?
We can easily add more mime-types if needed, now or in a later PR.